quantumdb-postgresql/src/main/java/io/quantumdb/core/planner/MigratorFunction.java

```diff
@@ -12,7 +12,7 @@
 	}
 
 	private final String name;
-	private final List<String> parameters;
+	private final Map<String, String> parameters;
```

Comment:
You probably always want this to be a `LinkedHashMap` to guarantee ordering.

Reply:
I'll change it:thumbsup: